CHANGELOG.md

```diff
@@ -8,3 +8,4 @@
 * Add custom MOTD/welcome message after logging in to console (#15)
 * Add StackStorm infrastructure integration tests, ship with new `st2-integration-tests` executable available to user (#20)
 * Install Virtualbox guest additions (#22)
+* Add continuous integration (#19)
```

Comment:
This should come into `In Development` now, since the `v2.7.1-20180507` was already released